protocol/0028-GOVE-governance.md

```diff
@@ -131,6 +131,27 @@ If a proposal is created and later a different outcome is preferred by network p
 
 Which of these makes most sense will depend on the type of change, the timing of the events, and how the rest of the community votes for the initial proposal.
 
+## Cancelling a proposal
+
+A proposal can be cancelled under certain conditions by, and only by, the party which created the proposal. To avoid situations where the community may have acted upon proposals they "expected" to pass, a party can only cancel their proposal before a certain "cancellation threshold" is reached.
+
+This cancellation threshold is a fixed point before the proposal closing time. The duration of the period in which a proposal cannot be cancelled is defined by the network parameter `governance.proposal.cancellationThreshold` (a duration string defaulting to `24h0m0s`).
```

Comment:
The original issue wanted to make the cancelation period relative to th3e voting period, while the spec now makes it a constant period. This shouldn't be an issue, but might be a bit odd if we have votes with a very short voting time (e.g., validator-only votes for changing spam parameters). In that case the cancelation period could be longer than the voting period, which leads to edge cases (maybe add to the spec something like "if the voting period is shorter than the cancelation period, then the cancelation period is set to the duration of the voting period)

Reply:
I think this is the other way round, so you'd end up unable to cancel a short enough vote, as it's a threshold that says how long before the end you bvecome unable to cancel a proposal, not how long it can be cancelled for.